src/components/project/export/ProjectExport.vue

```diff
@@ -0,0 +1,201 @@
+<template>
+    <div>
+        <v-unipept-card class="mb-6">
+            <v-card-title class="text-h5 font-weight-bold mb-2">
+                🧬 Export Project Overview
+            </v-card-title>
+            <v-card-text>
+                <p>
+                    You’re about to export your entire project. This includes all samples, groups, analysis results, and settings.
+                    The export will be saved as a <strong>.zip</strong> file that can later be re-imported or shared with others.
+                </p>
+            </v-card-text>
+        </v-unipept-card>
+
+        <v-alert
+            v-if="finishedAnalyses < amountOfSamples || peptonizerJobs > 0"
+            type="warning"
+            variant="tonal"
+            class="mb-6"
+            icon="mdi-alert-circle-outline"
+        >
+            <div class="d-flex justify-space-between align-center">
+                <span>
+                    <b>Warning:</b> Not all samples have been analyzed yet <b>({{ finishedAnalyses }} out of {{ amountOfSamples }} are finished)</b> OR
+                    some peptonizer jobs are still running <b>({{ peptonizerJobs }} {{ peptonizerJobs === 1 ? 'job' : 'jobs' }} running)</b>.
+                    Exporting your data now may result in incomplete results.
+                    Unfinished analyses will be analysed upon importing the project. The latest version of UniProtKB will be used for
+                    the analysis, rather than the version used in this project.
+                </span>
+
+            </div>
+        </v-alert>
+
+        <v-row class="mb-6" dense>
+            <v-col cols="12" sm="3">
+                <v-unipept-card class="d-flex flex-row py-2 px-4 align-center">
+                    <v-icon icon="mdi-account-group-outline" size="36" class="me-4" color="primary"/>
+                    <div>
+                        <div class="text-subtitle-1 font-weight-medium">Groups</div>
+                        <div class="text-h5 font-weight-bold">{{ amountOfGroups }}</div>
+                    </div>
+                </v-unipept-card>
+            </v-col>
+
+            <v-col cols="12" sm="3">
+                <v-unipept-card class="d-flex flex-row py-2 px-4 align-center">
+                    <v-icon icon="mdi-flask-outline" size="36" class="me-4" color="primary"/>
+                    <div>
+                        <div class="text-subtitle-1 font-weight-medium">Samples</div>
+                        <div class="text-h5 font-weight-bold">{{ amountOfSamples }}</div>
+                    </div>
+                </v-unipept-card>
+            </v-col>
+
+            <v-col cols="12" sm="3">
+                <v-unipept-card class="d-flex flex-row py-2 px-4 align-center">
+                    <v-icon icon="mdi-dna" size="36" class="me-4" color="primary"/>
```

Comment:
I think we should go for another icon then `dna` here :P This might be confusing since we're working with proteins. We can add a custom icon to our own Unipept Icon font (I'll be able to help you with this).